playfab-docs/features/data/dataconnections/index.md

```diff
@@ -27,6 +27,10 @@ You can get started with data connections for event ingestion using PlayFab’s
 
 Data connections is the replacement for the Event Export and Export to S3 features that currently PlayFab supports. Data connections allow for faster export in preferred formats for offline processing. You get to bring your storage account for data to be ingested and distributed. That means you get to manage and control the encryption-at-rest policy, the lifetime management policy, and network access. The billing becomes simplified as the costs associated with that storage account are accounted for centrally in your all-up Azure account billing.  
 
+### Blob Data Connection
```

Comment:
Is this header correct? This section seems to apply to data connections in general.

Reply:
Correct.